app/config.py

```diff
@@ -87,6 +87,7 @@ class Config(object):
             "tts-benefits-studio@gsa.gov",
         ],
     }
+    FEATURE_BEST_PRACTICES_ENABLED = getenv("FEATURE_BEST_PRACTICES_ENABLED", "false")
```

Comment:
Nice, and you have the default set if it's not found!

Out of curiosity, why a string `"false"` instead of Python's `False` boolean value?

Reply:
If this is because environment variables all come in as strings, let's handle that conversion here so we're dealing with real `True`/`False` values going forward:

```suggestion
    FEATURE_BEST_PRACTICES_ENABLED = bool(getenv("FEATURE_BEST_PRACTICES_ENABLED", False))
```

Furthermore, you could then use `0` or `1` for the environment variable value, like we do with the `REDIS_ENABLED` environment variable.

Reply:
So apparently a non-empty string value in Python converts to True when cast to a boolean using the bool() function. So might have to stick with `FEATURE_BEST_PRACTICES_ENABLED = getenv("FEATURE_BEST_PRACTICES_ENABLED", "false") == "true"
`